examples/step-89/step-89.cc

```diff
@@ -964,7 +985,7 @@ namespace Step89
                                               EvaluationFlags::values,
                                               /*sum_into_values=*/false);
                   // Don't zero out values again to keep already integrated
-                  // values
+                  // values PM: what do you mean? Where have you integrated before?
```

Comment:
`velocity_m_mortar.integrate()` writes values to the buffer. If `sum_into_values=false` the whole buffer, it zeroed out before writing the values. This means that at this point the values in `buffer` that correspond to the pressure are already zeroed out and `pressure_m_mortar.integrate()` can add the values into the buffer. If `sum_into_values=false` in this call, the values that have been written by `velocity_m_mortar.integrate()` are overwritten.

Reply:
OK. This is pure spaghetti code^^ `velocity_m_mortar.integrate(false)` also zeroes out the values of the pressure. But let's keep it like this for now.

Reply:
@peterrum I know :/ I thought there is a reason why the function zeroes out the whole buffer. Do you think it would make sense if `phi.integrate(false)` only zeroes out the values it will write to? In this case, I would add a TODO which can be resolved at some point.